presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -410,6 +410,17 @@ private Type handleResolvedField(Expression node, FieldId fieldId, Type resolved
             return setExpressionType(node, resolvedType);
         }
 
+        private RelationType getSourceRelation(Scope scope, RelationId relationId)
```

Comment:
I don't think it's needed, see my comment on https://github.com/prestodb/presto/pull/10375/files/098ddf08b4bfb2061fc3d855a307f5d14098ff5b..9949b16ce62d2a0c9b09291e3a2546cb3b01ef6b#diff-de0c3bd1446c0ac0bf16a9e1c91b20f5L403